src/pages/news/news-add/news-add.js

```diff
@@ -183,6 +184,17 @@ const NewsAdd = () => {
               required
             />
           </Paper>
+          <div className={classes.controlsBlock}>
+            <Link to={config.routes.pathToNews}>
+              <StandardButton
+                  id='back'
+                  title='Назад'
+                  variant='outlined'
+                  onClickHandler={() => {}}
+                  data-cy='back-btn'
+              />
+            </Link>
```

Comment:
Please move title to config, and set link to onclickHandler

Reply:
done